.github/workflows/test.yaml

```diff
@@ -28,6 +28,33 @@ jobs:
       - id: start-time-output
         run: echo "::set-output name=start-time::$(date)"
 
+  # From https://github.com/marketplace/actions/skip-duplicate-actions
+  # This action cleans up previously running instances of a workflow on the same branch.
+  # This accomplishes the task of automatically cancelling CI runs on pushes to the same branch,
+  # which is a common feature in most CI systems but currently not possible with GitHub actions.
+  cleanup-runs:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    if: "!startsWith(github.ref, 'refs/tags/') && !contains(github.actor, 'renovate') && github.ref != 'refs/heads/dev' && github.ref != 'refs/heads/integration' && github.ref != 'refs/heads/patch-dev' && github.ref != 'refs/heads/latest'"
```

Comment:
Why not allow this on renovate?

Reply:
Is there a way to skip this? On branches one might want to try a few things in quick succession by committing the variants and then look at all the CI results.

Reply:
You mean :trollface:  '!contains(github.actor, 'janpio')`? 

Did a copy paste, not sure we disabled it on renovate, it would be useful to have it, especially for renovate 😄

Reply:
it's now commented and not running